drivers/wifi/nrf700x/osal/fw_if/umac_if/inc/radio_test/fmac_structs.h

```diff
@@ -0,0 +1,91 @@
+/*
+ * Copyright (c) 2022 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+ */
+
+/** @file
+ *
+ * @addtogroup nrf700x_fmac_api_radio_test FMAC radio test API
+ * @{
+ *
+ * TODO: This file is not added doxygen to avoid duplicate warnings.
+ *
+ * @brief Header containing declarations for utility functions for
+ * FMAC IF Layer of the Wi-Fi driver.
+ */
+
+#ifndef __FMAC_STRUCTS_H__
+#define __FMAC_STRUCTS_H__
+
+#include <stdbool.h>
+
+#include "osal_api.h"
+#include "host_rpu_umac_if.h"
+#if !defined(__DOXYGEN__)
+#include "fmac_structs_common.h"
+#endif
+
+/**
+ * @brief  Structure to hold context information for the UMAC IF layer.
+ *
+ * This structure maintains the context information necessary for the
+ * operation of the UMAC IF layer.
+ */
+struct wifi_nrf_fmac_priv {
+	/* To fix duplicate warning */
+#if !defined(__DOXYGEN__)
+	/** Handle to the OSAL layer. */
+	struct wifi_nrf_osal_priv *opriv;
+	/** Handle to the HAL layer. */
+	struct wifi_nrf_hal_priv *hpriv;
+#endif /* !defined(__DOXYGEN__) */
+};
+
+
+/**
+ * @brief  Structure to hold per device context information for the UMAC IF layer.
+ *
+ * This structure maintains the context information necessary for the
+ * a single instance of an FullMAC based RPU.
+ */
+struct wifi_nrf_fmac_dev_ctx {
+	/* To fix duplicate warning */
+#if !defined(__DOXYGEN__)
+	/** Handle to the FMAC IF layer. */
+	struct wifi_nrf_fmac_priv *fpriv;
```

Comment:
this looks like a hack, why is it done like this?

Reply:
Yes, it is kind of a hack, doxygen complains about `duplicate declaration` even for members, so, in order to properly fix I have to rename all members and their usage in source code with proper ifdef's, as it's not adding much value, just removed these structures.